server/controllers/PartyController.js

```diff
@@ -18,6 +18,28 @@ class PartyController {
     }
     return error(response, 400, 'Please enter the missing fields');
   }
+
+  static deleteParty(request, response) {
+    const { id } = request.params;
+
+    const parseId = parsedInt(id);
+
+    // check if id is a number
+    if (!(Number.isInteger(parseId))) {
+      return error(response, 404, '404', 'Sorry, the party id must be an integer');
```

Comment:
It is good practice you extract validations into separate file

Reply:
ohkay thanks i will do